packages/scandipwa/src/component/CartOverlay/CartOverlay.component.js

```diff
@@ -210,6 +210,7 @@ export class CartOverlay extends PureComponent {
                   elem="CartButton"
                   mix={ { block: 'Button', mods: { isHollow: true } } }
                   to="/cart"
+                  onClick={ window.scrollTo({ top: 0 }) }
```

Comment:
I guess, this doesn't work. Since you call `window.scrollTo` on component render and apply it's result to `onClick` prop.